### DISCUSSION TITLE
Using with Virtual DJ

### DISCUSSION THREAD

**Discussion by stewartmyles** [cpyarger/obs-midi#119] [Q&A] [answered]
I'm looking to use the OBS midi plugin to enable transitions and effects such as the spectral plugin, it is working perfectly and doing what I require however I want to use this plugin in conjunction with an application called VirtualDJ this also uses midi for the input triggers. I have two midi devices I want to use one for virtual dj and one for OBS control. With OBS running both the midi controllers are locked exclusively to OBS even if I untick the device in OBS midi settings, I have noticed that even if I untick the device enable it becomes enabled again. Is this a bug or am I doing something wrong.
Any help would really be appreciated

**Comment by cpyarger** [comment]
I am taking a look into it

**Comment by cpyarger** [comment, marked as answer]
This is a bug, you can go into your config file and manually disable whichever device you need to. by switching ```"enabled" : true``` to ```"enabled" : false```
